tools/github_helper/main.go

```diff
@@ -93,16 +93,42 @@ func newHelper(r *string) (*helper, error) {
 	}
 }
 
+// Create a new branch for fast forward
+func (h helper) createBranchForFastForward(commit *string) error {
+	branchName := fmt.Sprintf("fastForward-%s", (*commit)[0:7])
+	ref := fmt.Sprintf("refs/heads/%s", branchName)
+	gho := github.GitObject{
+		SHA: commit,
+	}
+	r := github.Reference{
+		Ref:    &ref,
+		Object: &gho,
+	}
+
+	if _, _, err := h.Client.Git.CreateRef(context.TODO(), h.Owner, h.Repo, &r); err == nil {
+		return nil
+	} else {
+		return err
+	}
+}
+
 // Creates a pull request from Base branch
 func (h helper) createPullRequestToBase(commit *string) error {
 	if commit == nil {
 		return errors.New("commit cannot be nil.")
 	}
+
+	if err := h.createBranchForFastForward(commit); err != nil {
```

Comment:
We should probably check if the branch already exists and return nil in that case.